_static/files/audit-event-examples/R4_read.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
```

Comment:
System changed in R4, needs to be "http://terminology.hl7.org/CodeSystem/audit-event-type"